compiler/z/codegen/ConstantDataSnippet.hpp

```diff
@@ -47,7 +47,7 @@ class S390ConstantDataSnippet : public TR::Snippet
    protected:
    union
       {
-      uint8_t                       _value[16];
+      uint8_t                       _value[64];
```

Comment:
Can this be `1 << TR_DEFAULT_DATA_SNIPPET_EXPONENT` so as to link the two entities together. I also question the value of `TR_DEFAULT_DATA_SNIPPET_EXPONENT` to begin with. Is there any possibility to simplify this code? In what context is `constantDataSnippetExponent` used?